src/abi/wrapperAbi.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "constant": true,
+    "inputs": [
+      {
+        "name": "_owner",
+        "type": "address"
+      }
+    ],
+    "name": "balanceOf",
+    "outputs": [
+      {
+        "name": "balance",
+        "type": "uint256"
+      }
+    ],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  },
```

Comment:
because we use `useBalance` hook we _can_ remove this. But can also stay for future possible purposes.

Reply:
Leaving it, just in case